packages/synapse-react-client/src/components/download_list/FolderDownloadConfirmation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useSynapseContext } from '../../utils'
+import {
+  useAddQueryToDownloadList,
+  useGetEntityChildren,
+} from '../../synapse-queries'
+import { displayToast } from '../ToastMessage'
+import { DownloadConfirmationUI } from './DownloadConfirmationUI'
+import { displayFilesWereAddedToDownloadListSuccess } from './DownloadConfirmationUtils'
+import { EntityType } from '@sage-bionetworks/synapse-types'
+
+export type FolderDownloadConfirmationProps = {
+  folderId: string
+  fnClose: () => void
+}
```

Comment:
Keep same prop signature as original DownloadConfirmation as this is used in SWC